utils/debug/Debug.js

```diff
@@ -279,5 +272,90 @@ function Debug(scene) {
             }
         };
     })());
+
+    // hook that to scene.update
+    const oldUpdate = Object.getPrototypeOf(scene).update.bind(scene);
```

Comment:
It's a little tricky
Need to add two custom functions in scene: preUpdate () and postUpdate ()